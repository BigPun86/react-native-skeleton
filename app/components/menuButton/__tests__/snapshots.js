import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Component from '../index.js';

it('renders correctly without props', () => {
  const tree = renderer.create(<Component/>).toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders correctly with props', () => {
  const view = (
    <Component onPress={()=>{}} icon={'gear'}/>
  );
  const tree = renderer.create(view).toJSON();
  expect(tree).toMatchSnapshot();
});
