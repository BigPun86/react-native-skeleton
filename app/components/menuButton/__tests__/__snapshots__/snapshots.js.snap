exports[`test renders correctly without props 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  hitSlop={undefined}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Array [
      Object {
        "backgroundColor": "transparent",
      },
      Object {
        "paddingHorizontal": 15,
        "paddingVertical": 10,
      },
    ]
  }
  testID={undefined}>
  <Text
    accessible={true}
    allowFontScaling={false}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "color": "white",
          "fontFamily": "FontAwesome",
          "fontSize": 24,
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
        undefined,
      ]
    }>
    ï€“
  </Text>
</View>
`;
